app/scripts/controllers/main.js

```diff
@@ -93,6 +108,27 @@ app.controller('MainCtrl', [
       $scope.autoShow = !$scope.autoShow
     }
 
+    $scope.enableNotifications = function () {
+      if (window.Notification && window.Notification.permission === 'granted') {
+        window.alert('To disable notifications, revoke the permissions in your browser.')
+        return
+      }
+      window.Notification.requestPermission().then(function (permissions) {
+        $scope.webNotifications = permissions === 'granted'
+      }).catch(function () {
+        window.alert('Unable to enable web notifications. See console for more information')
+      })
+      if (!window.isSecureContext && window.console) {
+        console.info(
```

Comment:
I'd rather make this more obvious to the user, perhaps using window.alert or a `title` on the html element.